src/app.ts

```diff
@@ -0,0 +1,61 @@
+import { Command, program } from "commander"
+import * as path from "path"
+import chalk from "chalk"
+import {
+  log,
+  getVersion,
+  expectPort,
+  Ports,
+  copyTemplate,
+  updateTemplate,
+} from "./helpers"
+import { run_yarn } from "./cmd"
+
+export async function run(args: string[], branch_name: string) {
+  program
+    .version(getVersion())
+    .command("create <project_name>", { isDefault: true })
+    .description("Make Something Original")
+    .option(
+      "-b, --backend-port <port_number>",
+      "dev backend API port",
+      expectPort,
+    )
+    .option(
+      "-f, --frontend-port <port_number>",
+      "dev frontend server port",
+      expectPort,
+    )
+    .option("-d, --db-port <port_number>", "dev database port", expectPort)
+    .option("-n, --without-yarn", "Do not run yarn in project dir")
```

Comment:
weirdly, `--no-run-yarn` gets turned into `opts.runYarn` by commander, so I opted for `--without-yarn`.